### ISSUE TITLE
back button doesn't work yet

### ISSUE THREAD

**Issue by zaceno** [mrozbarry/hyperapp-router#1]
The back button doesn't work yet. I know it's early days for this library but in the past I know you've appreciated having reports like this in the github issue tracker so you have a checklist to work off of. :)

**Comment by zaceno** [comment]
Unfortunately, even after b01df0508eb941dc6c96a82f00c264aaec1389c4 (popstate support) it seems the back and forward buttons still don't work reliably.

**Comment by mrozbarry** [comment]
Does that mean occasionally the back button updates the url as expected, but not the app, or something else?

**Comment by zaceno** [comment]
@mrozbarry correct. Mostly that is the case. It’s weird that it isn’t consistently working or not working, but I didn’t spend enough time on it to figure out when it works correctly (I only noticed once or twice that the app was updating in response to the back button but I didn’t figure out what was special about those times)

I did observe the bug in both Safari and Chrome. I’m going to test it some more to get to the bottom of it.
